config/environments/production.rb

```diff
@@ -32,21 +27,23 @@
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
-  # Generate digests for assets URLs.
-  config.assets.digest = true
```

Comment:
new defaults said it's no longer needed ...   page renders with  `<script src="/assets/application-fd416841b5adf3d3928cc66d8fc34e880ee4c38a8abb0154706b716096750bfc.js"></script>` and it works ...